### PR TITLE
Fix/geolocation mockdata

### PR DESCRIPTION
**Pull request by ranjan-stha**
## Changes

* Updated the mock data related to geolocations
* Postprocess the geolocation response 

Mention related users here if any.

## This PR doesn't introduce any:

- [X] temporary files, auto-generated files or secret keys
- [X] n+1 queries
- [X] flake8 issues
- [X] `print`
- [X] typos
- [ ] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: analysis_module/mockserver.py [hunk 1]
```diff
@@ -213,9 +212,19 @@ def process_geolocation(body) -> Any:
...
     def shape_geo_entities(entity: dict, excerpt: str):
-        ent = entity.copy()
-        start = random.randint(0, len(excerpt) - len(ent["ent"]))
-        ent.update({"offset_start": start, "offset_end": start + len(ent["ent"])})
+        ent = {}
+        ent["entity"] = entity["ent"]
+        start = random.randint(0, len(excerpt) - len(entity["ent"]))
+        ent["meta"] = {}
+        ent["meta"].update({"offset_start": start, "offset_end": start + len(entity["ent"])})
+        ent["meta"].update({"latitude": None, "longitude": None})
```
Comment (Rup-Narayan-Rajbanshi) [inline analysis_module/mockserver.py]:
```suggestion
          ent["meta"].update(
        {
            "offset_start": start,
            "offset_end": start + len(entity["ent"]),
            "latitude": None,
            "longitude": None
        }
    )

```

File: analysis_module/mockserver.py [hunk 2]
```diff
@@ -262,7 +267,7 @@ def shape_geo_entities(entity: dict, excerpt: str):
 
 
 def geolocation_mock_model(body) -> Any:
-    process_geolocation.delay(body, countdown=2)  # Trigger task after 2 seconds
+    process_geolocation.delay(body)  # Trigger task after 2 seconds
```
Comment (Rup-Narayan-Rajbanshi) [inline analysis_module/mockserver.py]:
```suggestion
    process_geolocation.delay(body)
```

**Review by Rup-Narayan-Rajbanshi — CHANGES_REQUESTED** [review]

File: analysis_module/mockserver.py (hunk 1 above)
Comment (ranjan-stha) [reply analysis_module/mockserver.py]: @Rup-Narayan-Rajbanshi done

File: analysis_module/mockserver.py (hunk 2 above)
Comment (ranjan-stha) [reply analysis_module/mockserver.py]: @Rup-Narayan-Rajbanshi done.

**Review by sudan45 — APPROVED** [review]

**Review by Rup-Narayan-Rajbanshi — APPROVED** [review]

**Pull request merged**